FRW/Modules/Login/ImportAccountsView.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  ImportAccountsView.swift
+//  FRW
+//
+//  Created by cat on 2024/8/19.
+//
+
+import SwiftUI
+import SwiftUIX
+
+class ImportAccountsViewModel: ObservableObject {
```

Comment:
nit: unless you know a class will be subclassed, it's always better to declare as `final` for a small performance gain: Inheritable classes use a vtable for method resolutions, causing invocations to be resolved at runtime. In a final class instead there's no vtable and method invocations are resolved at compilation time.